src/ontology/dpo-odk.yaml

```diff
@@ -19,20 +19,44 @@ export_formats:
   - obo
   - json
 import_group:
+  use_base_merging: TRUE
+  exclude_iri_patterns:
+    - <http://purl.obolibrary.org/obo/UBERON_*>
+    - <http://purl.obolibrary.org/obo/OBI_*>
+    - <http://purl.obolibrary.org/obo/SO_*>
+    - <http://purl.obolibrary.org/obo/NCBITaxon_*>
   products:
     - id: chebi
-      use_gzipped: TRUE
-      module_type: minimal
+      make_base: TRUE
+      mirror_from: https://raw.githubusercontent.com/obophenotype/chebi_obo_slim/main/chebi_slim.owl
     - id: fbdv
+      use_base: TRUE
     - id: fbbt
+      make_base: TRUE
```

Comment:
(while you dont do that for fbdv)

Reply:
The problem is that FBbt, being maintained in OBO format, defines “shorthands” for several RO relations in its edit file, such as:

```
[Typedef]
id: part_of
name: part_of
xref: BFO:0000050
```

(Mostly for the benefit of being able to write `part_of` instead of `BFO:0000050` in the rest of the file; not entirely convinced this is worth the trouble, but that’s what we do, and as the OBO Flat File Format specification [writes](https://owlcollab.github.io/oboformat/doc/obo-syntax.html#8), “the use of symbolic identifiers such as 'part_of' is common in almost all OBO format ontologies, and has a precedent stretching back over ten years”.)

Those IDs and labels end up in the ODK-generated “base” file for FBbt, since they are in the -edit file and currently what the ODK calls “base” is merely the contents of the -edit file.

If we used FBbt’s “base” here, the re-defined label would clash with the real label of those relations that we get from RO.

So instead, we make an (arguably) “real” base here at mirroring time, by stripping everything that’s not in the FBbt namespace.

And I would have done the same for FBdv (for the same reason), if not for the fact that the `robot remove` command the ODK is using for `make_base` has the side-effect of removing transitivity declarations from annotation properties.

I fully agree that it would be a better solution to have FBbt (and FBdv) to generate “real” base files, _without_ any re-defined RO labels. But the ODK doesn’t do that, and for now I’d rather stick to what the ODK is doing.